User.java

```diff
@@ -0,0 +1,74 @@
+
+public class User {
+	/*For the User class the fields needed are:
+	 * Name, age, city, state, dissabilities, and
+	 * number of families.
+     * 
+     * I added simple setters and getters and some boolean methods 
```

Comment:
For a shared codebase with multiple collaborators, phrases like "I added ___" in comments can quickly get confusing, as multiple people may edit this class at some point. Phrasing this in a more neutral way (perhaps "class also contains ___ for doing ____" or similar) can help the comment remain clear as time goes on.

Alternatively, this type of comment is something I often add in the body of the commit message, especially if you include details on why you feel this is the best place for those methods, etc. (Might be overkill in this situation, but overkill is underrated.)